### PR TITLE
Remove TranscribedLinesConnector

### PR DESCRIPTION
**Pull request by eatyourgreens**
`TranscribedLinesConnector` can be removed, and replaced by the `useTranscriptionReductions` hook in the `TranscribedLines` component.

All it does is run the hook and pass the returned object down as component props. It's been kept separate because that made the Enzyme tests easier to write.

_Please request review from `@zooniverse/frontend` team or an individual member of that team._ 

## Package
lib-classifier

## How to Review
This can be tested with the `TranscribedLines` story in the storybook. It shouldn't change the transcription task at all.

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [x] Tests are passing locally and on Github
- [x] Documentation is up to date and changelog has been updated if appropriate
- [x] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
- [ ] All pages of a FEM project load: Home Page, Classify Page, and About Pages
- [ ] Can submit a classification
- [ ] Can sign-in and sign-out
- [ ] The component is accessible
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## Refactoring
- [x] The PR creator has described the reason for refactoring
- [x] The refactored component(s) continue to work as expected

### CODE REVIEW

**Comment by eatyourgreens** [comment]
Setting this to draft because component props are broken in the tests.

File: packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.spec.js [hunk 1]
```diff
@@ -279,101 +218,94 @@ describe('Component > TranscribedLines', function () {
...
+        expect(line.getAttribute('aria-disabled')).to.equal('true')
+        expect(line.tabIndex).to.equal(-1)
+        // expect(task.marks.length).to.equal(1)
+//         expect(task.activeMark).to.exist()
+//         expect(task.activeMark.x1).to.equal(transcribedLines[0].points[0].x)
+//         expect(task.activeMark.y1).to.equal(transcribedLines[0].points[0].y)
+//         expect(task.activeMark.x2).to.equal(transcribedLines[0].points[1].x)
+//         expect(task.activeMark.y2).to.equal(transcribedLines[0].points[1].y)
+//         expect(task.activeMark.frame).to.equal(transcribedLines[0].frame)
+//         task.setActiveMark(undefined)
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.spec.js:253]: I can't see why these tests fail. If I inspect `task.marks` inside the `TranscribedLines` component, after the line is clicked, it contains a new mark, which is active. If I inspect `task.marks` here, in the tests, its still an empty array.

Comment (eatyourgreens) [reply packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.spec.js]: These tests are clicking a disabled line, which does nothing.

**Review by kieftrav — APPROVED** [review]
Pretty straightforward - LGTM!

**Pull request merged**